packages/blockchain-wallet-v4-frontend/src/modals/IdentityVerification/IdentityVerification.spec.js

```diff
@@ -94,84 +96,97 @@ describe('IdentityVerification Modal', () => {
-      //   wrapper
-      //     .find('Field[name="dob"]')
-      //     .find('SelectBox')
-      //     .prop('input')
-      //     .onChange('11')
-      //   wrapper
-      //     .find('Field[name="dob"]')
-      //     .find('input[name="year"]')
-      //     .simulate('change', {
-      //       target: { value: '1999' }
-      //     })
-      //   wrapper
-      //     .find('Field[name="lastName"]')
-      //     .find('input')
-      //     .simulate('change', { target: { value: 'Beloved' } })
-      //   wrapper
-      //     .find('Field[name="firstName"]')
-      //     .find('input')
-      //     .simulate('change', { target: { value: 'User' } })
-      //   jest.runAllTimers()
-      //   wrapper.update()
-      //   expect(wrapper.find('Button[type="submit"]').prop('disabled')).toBe(
-      //     false
-      //   )
-      // })
-
-      // it('should validate age to be over 18', () => {
-      //   wrapper
-      //     .find('Field[name="dob"]')
-      //     .find('input[name="date"]')
-      //     .simulate('change', {
-      //       target: { value: `11` }
-      //     })
-      //   wrapper
-      //     .find('Field[name="dob"]')
-      //     .find('SelectInputContainer')
-      //     .prop('onChange')('11')
-      //   wrapper
-      //     .find('Field[name="dob"]')
-      //     .find('input[name="year"]')
-      //     .simulate('change', {
-      //       target: { value: new Date().getFullYear() - 17 }
-      //     })
-      //   wrapper
-      //     .find('Field[name="lastName"]')
-      //     .find('input')
-      //     .simulate('change', { target: { value: 'Beloved' } })
-      //   wrapper
-      //     .find('Field[name="firstName"]')
-      //     .find('input')
-      //     .simulate('change', { target: { value: 'User' } })
-      //   jest.runAllTimers()
-      //   wrapper.update()
-      //   expect(wrapper.find('Button[type="submit"]').prop('disabled')).toBe(
-      //     true
-      //   )
-      // })
+      it('should validate age to be over 18', () => {
+        wrapper
+          .find('Field[name="dob"]')
+          .find('input[name="date"]')
+          .simulate('change', {
+            target: { value: `11` }
+          })
+        wrapper
+          .find('Field[name="dob"]')
+          .find('SelectInputContainer')
+          .prop('onChange')('11')
+        wrapper
+          .find('Field[name="dob"]')
+          .find('input[name="year"]')
+          .simulate('change', {
+            target: { value: '2005' }
+          })
+        wrapper
+          .find('Field[name="lastName"]')
+          .find('input')
+          .simulate('change', { target: { value: 'Beloved' } })
+        wrapper
+          .find('Field[name="firstName"]')
+          .find('input')
+          .simulate('change', { target: { value: 'User' } })
+        jest.runAllTimers()
+        wrapper.update()
+        expect(wrapper.find('Button[type="submit"]').prop('disabled')).toBe(
+          true
```

Comment:
all the fields that lead to button being enabled should be filled, otherwise it will be disabled because of incomplete fields
also we can check for an error message